src/sqlancer/cnosdb/CnosDBProvider.java

```diff
@@ -69,9 +69,10 @@ public CnosDBConnection createDatabase(CnosDBGlobalState globalState) throws Exc
         CnosDBClient client = new CnosDBClient(host, port, username, password, databaseName);
         CnosDBConnection connection = new CnosDBConnection(client);
         client.execute("DROP DATABASE IF EXISTS " + databaseName);
+        String createdb = "CREATE DATABASE " + databaseName + " WITH TTL '1000000' REPLICA 2";
```

Comment:
What does `WITH TTL '1000000'` mean? Do you think it might be useful to add a short comment why we override the default configuration?